### ISSUE TITLE
Error during `terraform plan`

### ISSUE THREAD

**Issue by elliotcm** [CrunchyData/terraform-provider-crunchybridge#14]
Hello,

When running `terraform plan` I'm getting the following error:
```
│ Error: error unmarshaling response body (account detail): json: cannot unmarshal bool into Go value of type []bridgeapi.Team
│
│   with module.database.data.crunchybridge_account.user,
│   on ../../modules/database/main.tf line 20, in data "crunchybridge_account" "user":
│   20: data "crunchybridge_account" "user" {}
```
The line in question is just that,:
```
data "crunchybridge_account" "user" {}
```
I've tried deleting the `.terraform` directory and re-running `terraform init`, but same result when running `terraform plan`.

This was working a couple weeks ago and nothing in that `.tf` file has changed in 6 months, including the version of this plugin, however I *did* update macos this morning so I'm wondering if that's related, but the error's too internal for me to follow along any further.

Terraform / plugin versions:
```
Terraform v1.8.5
on darwin_arm64
+ provider registry.terraform.io/crunchydata/crunchybridge v0.2.0
+ provider registry.terraform.io/hashicorp/aws v5.56.1
+ provider registry.terraform.io/hashicorp/tls v4.0.5
```
MacOS version `15.1.1 (24B91)`.

**Comment by wilybrace** [comment]
Thanks for the report, it does appear at first glance to be an internal issue, I'll make sure that it is addressed in the outgoing release.

**Comment by dentarg** [comment]
Yeah, pretty mind boggling to have the example in the ~~README~~ [registry](https://registry.terraform.io/providers/CrunchyData/crunchybridge/latest/docs) not work out-of-the-box? Not a lot CrunchyData/CrunchyBridge Terraform users out there?

I removed `data "crunchybridge_account" "user" {}` and for the `crunchybridge_cluster` resource that wants `team_id` I used the team ID from the dashboard URL `https://crunchybridge.com/teams/<team ID>/dashboard` (in the example it is `data.crunchybridge_account.user.default_team`).

**Comment by wilybrace** [comment]
I have confirmed that the Terraform Registry has now synchronized with the latest bugfix release, `v0.3.0`.
